core/templates/filters/string-utility-filters/filter-for-matching-substring.pipe.spec.ts

```diff
@@ -0,0 +1,67 @@
+// Copyright 2019 The Oppia Authors. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS-IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+/**
+ * @fileoverview Tests for CamelCaseToHyphens pipe for Oppia.
+ */
+
+import { FilterForMatchingSubstringPipe } from
+  'filters/string-utility-filters/filter-for-matching-substring.pipe';
```

Comment:
You can move this to the above line itself.

Reply:
Could you clarify, do you mean move it one line up or above the `@fileoverview`?

Reply:
import statements are exempted from max-len violations. What @nithusha21 might have meant was to have the entire import in one line.

Reply:
Got it, done